src/test/resources/MakeAppointmentConfirm.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Make Appointment
+
+  An a user
+  I want to sign in
+  So I can use the app
+  I make Appointment
+
+  @automated
+  Scenario Outline: User can make appointment with valid value
+    Given user is in Katalon login pages
```

Comment:
xóa step này đi và thay bằng:    ` Given user is in Katalon login page`, đã tồn tại step làm hành động tương tự thì không được sinh thêm step mới, mất thời gian